docs/rules/no-dispatch-in-effects.md

```diff
@@ -1 +1,60 @@
 # no-dispatch-in-effects
+
+> This rule disallows dispatching actions as side effects in NgRx Effects.
+
+## Rule Details
+
+An effect should handle actions and map it to a singular action. Each effect should be clear and concise and must be readily understandable as to what it does affect. Dispatching from inside an effect instead (or together with) mapping it to a new a action can result in painfully hard-to-find bugs and is generally considered a bad practice.
```

Comment:
```suggestion
An effect should handle actions and map them to a singular action. Each effect should be clear and concise and must be readily understandable as to what it does affect. Dispatching from inside an effect instead of (or together with) mapping it to a new a action can result in painfully hard-to-find bugs and is generally considered a bad practice.
```